src/templates/base.html

```diff
@@ -33,18 +37,12 @@
     </div>
     {% set view_args = request.view_args.copy() %}
```

Comment:
the declaration in line 28 is sufficient ?

Reply:
Resolved this by pulling out the view_args logic into a Python function that we can call from the templates.

Reply:
Great !